src/internal/components/dropdown/index.tsx

```diff
@@ -173,8 +173,6 @@ const Dropdown = ({
   const triggerRef = useRef<HTMLDivElement | null>(null);
   const dropdownRef = useRef<HTMLDivElement | null>(null);
   const dropdownContainerRef = useRef<HTMLDivElement | null>(null);
-  // This container is only needed to apply max-height to. We can't move max-height to it's parent
-  // because of an IE11 issue with flexbox. https://github.com/philipwalton/flexbugs/issues/216
   const verticalContainerRef = useRef<HTMLDivElement>(null);
```

Comment:
I tried to refactor this part of code and it turned out that the purpose of this container not only to apply `max-height`. Once I got rid of it and applied the styles to its parent, things fell apart, so I didn't find it beneficial to refactor this part of code and decided to remove the comment, because it's does not reflect the purpose of this container